.gitignore

```diff
@@ -0,0 +1,3 @@
+.github/workflows/linters.yml
+node_modules/
+Test.md/
```

Comment:
Try to remove the slash from `Test.md` file, we add a slash for folders

Reply:
oh thanks